tweets/2021-11/postopfmeetup2.tweet

```diff
@@ -0,0 +1,3 @@
+Thanks to all who attended the Operate First Data Science Meetup. In case you missed it, please watch @paco_space talk about how to create & share reproducible notebooks with jupyterlab-requirements.
```

Comment:
```suggestion
In case you missed this week's Operate First Data Science Meetup, checkout @paco_space talk about how to create & share reproducible notebooks with jupyterlab-requirements.
```